pkg/handler/handler.go

```diff
@@ -1,6 +1,7 @@
 package handler
 
 import (
+	auth_handler "github.com/dwarvesf/fortress-api/pkg/handler/auth"
```

Comment:
Mình đang dùng alias ở đây, thì ở trong package handler/auth có thể rename thành package auth_handler ---> package auth

Reply:
em đang để auth_handler để consistent với package employee, metadata
chắc em đổi tên tat cả package handler cho consistent luôn không anh?

Reply:
@huynguyenh what do you think fen?

Reply:
@mkhoi1998 a nghĩ nên đổi lại.